<launch> 
        <arg name ="port"/>
        <node name="launcherd" pkg="gps_puck" type="driver.py" output="screen" >
         <param name="port_number" value="$(arg port)"/>
        </node>
</launch> 